packages/seed/src/commands/test/testCustomFixture.ts

```diff
@@ -24,6 +25,7 @@ export async function testCustomFixture({
     workspace: SeedWorkspace;
     generatorType: GeneratorType;
     irVersion: string | undefined;
+    outputVersion: string | undefined;
```

Comment:
looks like we always pass `undefined` here -- b/c we do that thoughts on removing the arg and just passing it as undefined to `acquireLocksAndRunTest` ?